internals/models.py

```diff
@@ -885,6 +909,48 @@ def getSortingMilestone(feature):
 
     return allowed_feature_list
 
+  @classmethod
```

Comment:
Now I see what you mean about basing one PR on another.   Since this diff is pretty big, I will need you to rebase rather than me trying to resolve a small conflict.

Reply:
Yup... I updated the main in my fork and then rebased this branch with the updated main branch again. Now, the diff of this PR shows changes made in this PR only.